spanner/google/cloud/spanner/_helpers.py

```diff
@@ -38,6 +38,7 @@ class TimestampWithNanoseconds(datetime.datetime):
     """
     __slots__ = ('_nanosecond',)
 
+    # pylint: disable=arguments-differ
```

Comment:
This is a nasty one. Does this feel like a bug in Pylint? (`disable` is fine as-is)